include/slg/textures/texture_funcs.cl

```diff
@@ -1185,3 +1185,27 @@ OPENCL_FORCE_INLINE float3 MakeFloat3Texture_ConstEvaluateSpectrum(__global HitP
 }
 
 #endif
+
+//------------------------------------------------------------------------------
+// Rounding texture
+//------------------------------------------------------------------------------
+
+#if defined(PARAM_ENABLE_TEX_ROUNDING)
+
+OPENCL_FORCE_NOT_INLINE float RoundingTexture_ConstEvaluateFloat(__global HitPoint *hitPoint,
+                                                                 const float tex1,
+                                                                 const float tex2) {
+    float innerBound = tex2 * (int) (tex1 / tex2);
```

Comment:
Possible division by zero if `tex2` is 0.
How should this case be handled? 
(if a complex texture is used as input it can be that some pixels are 0)